wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
-**Modding** is the process of examining another's [map](Beatmap "wikilink") and making suggestions. These suggestions can be either to fix errors or to simply improve the map. The person modding the map is called a "modder", and the person who made the map is referred to as the "mapper". Modding can be useful in three main ways:
+# Modding
 
-1.  If your mod post is useful to the mapper, then they will give you [kudosu](kudosu "wikilink"). Kudosu can be used to boost your map's star priority up.
-2.  "Mod4mods", or when you make a deal to mod somebody else's map if they mod yours, are a great way to get your map modded.
-3.  A good mod post reduces the amount of time the approval teams spend pointing out basic mistakes. This in turn makes the approval teams less stressed, and that in turn makes them more likely to mod your map. Trust us, happy [BNs](Beatmap_Nomination_Group "wikilink") are typically in a better mood than a BN who just spent 10 minutes crying because of a map that clearly needed a lot more modding.
+Modding is the process of examining another mapper's [beatmap](/wiki/beatmap) and giving constructive feedback to help improve it.
+The person modding a map is called a _modder_ while the person who creates the map is called a _mapper_.
+Modding can be useful in three main ways:
 
-If you are learning how to map or already know how to map, modding can be a great way to improve mapping skills.
+- If a modder's post is useful to the mapper, then the mapper may give the modder a [kudosu](/wiki/kudosu).
+  Kudosu can be used to boost a map's [star priority](/wiki/star_priority).
+- "M4M" (mod-for-mod); when a modder makes a deal to mod the map if the receiver mods the modder's map.
```

Comment:
is this rewording of your example ok?
`
Mod for Mod (M4M): the process where two mappers make a deal to mod each other's mapsets and thus exchange mods.
`